libsolidity/interface/CompilerStack.cpp

```diff
@@ -481,6 +481,14 @@ Json::Value const& CompilerStack::natspec(Contract const& _contract, Documentati
 	return *(*doc);
 }
 
+Json::Value CompilerStack::functionHashes(ContractDefinition const& _contract)
```

Comment:
Hm, the other version seems to have had a mapping signature -> hex - could you change that please?